src/content/docs/journey-app-slow/external-services.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Identify slow external services and dependencies
+metaDescription: Use the external services UI to identify slow external services in your app
+---
+
+## Objectives
+
+By the end of this tutorial, you will:
+
+* Understand how to use the external services UI
+* Have identified any slow external services such as API calls
+* Review transaction-level performance
+* Find traces that involve the transactions you're interested in
+
+## Why look at external services 
+
+The external services feature allows you to observe the upstream and downstream activity of a service. Those upstream or downstream external services may be your own services you instrumented, or more likely in this case third-party services and API calls. When these external services have a slow response time, they hold your app up causing slowness.
+
+
+## Identify slow external services
+
+Let's take a look at how to identify slow external services:
+
+<Steps>
+
+    <Step>
+         Navigate to the external services UI: Go to **[one.newrelic.com](https://one.newrelic.com/nr1-core) > (select an app) > Monitor > External services**. Toggle **Show new view** in the top left if not already enabled and then click **Map** in the top right.
+
+         Your application’s dependencies, also known as upstream dependencies, are listed on the left. On the right are services that depend on your application, known as downstream services. You can see which services are healthy as well as the throughput and time consumption for each service. You can select a service to see more detailed information on that service.
+
+         Hover over all your upstream dependencies, and try to identify any that have a high response time or trace count. Pay specific attention to any that have signifigant spikes in their recent history. If you find a suspicous service, drill in deeper by hovering over it and selecting the **View traces** button. This new pane displays information about that transactions trace, which records the available function calls, database calls, and external calls of that transaction. 
+
+        There are various ways to use the information provided by this page, so feel free to explore. Here's an example workflow:
```

Comment:
Nice

Reply:
unironic. i like the example workflow because it's like those training rails in bowling alleys